### DISCUSSION TITLE
protocol 'git@https' is not supported

### DISCUSSION THREAD

**Discussion by alexgenkin** [community/community#31085] [Other Feature Feedback, Questions, & Ideas]
fatal: protocol 'git@https' is not supported 
either VSC or Github Desktop

**Comment by jfullstackdev** [comment]
this might help 

https://stackoverflow.com/questions/53988638/git-fatal-protocol-https-is-not-supported

**Reply by alexgenkin** [reply]
> this might help
> 
> [stackoverflow.com/questions/53988638/git-fatal-protocol-https-is-not-supported](https://stackoverflow.com/questions/53988638/git-fatal-protocol-https-is-not-supported)

I read this before posting. Those are cases of erroneous command line typing. In my case there is no command line, just buttons on Github Desktop and VSC

**Comment by airtower-luna** [comment]
That looks like the remote URL for your repository is broken. `git@` is at the start of SSH URLs for GitHub, `https://` at the start of HTTPS URLs in general. You have to pick one or the other to access your repository.

You can find the right URL behind the green "Code" button on the repository page, select the protocol you want and copy the URL. You can change a remote URL with [`git remote set-url`](https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem) (or just do a fresh clone).

**Reply by punkfriend** [reply]
remember that time your 3 year old comment helped me after tons of unsuccessful google searching? 
Thank you github user airtower luna
